### PR TITLE
Menu Browsing Page  Front End

### PR DESCRIPTION
**Pull request by bbuffington1662**

### CODE REVIEW

**Review by cheddartot — APPROVED** [review]

File: app/templates/menu-browse.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+<html>
+    <head>
+        <title>{{ title }}</title>
+        <style>
+            .code {
+                background-color: #d8d1de;
+                color: #45474a;
+            }
+
+            .resturant-item {
```
Comment (pgayee694) [inline app/templates/menu-browse.html]: move styling to a css file

File: app/templates/menu-search.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>{{ title }}</title>
+        <style>
+
+            .code {
```
Comment (pgayee694) [inline app/templates/menu-search.html]: move styling to css file

File: app/templates/menu-search.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            <select name="categories" multiple>
+                {% for category in categories %}
+                <option value="{{ category }}">{{ category }}</option>
+                {% endfor %}
+            </select>
+			<br/>
+
+            <p>Cuisines</p>
+            <select name="cuisines" multiple>n     
+                {% for category in categories %}
```
Comment (pgayee694) [inline app/templates/menu-search.html]: should be iterating over cuisines

File: app/templates/menu-search.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+			.center {
+				margin-left: 0%;
+			}
+
+        </style>
+    </head>
+    <body class="code">
+        <h1>{{ title }}</h1>
+
+        <form action="" class="form">
```
Comment (pgayee694) [inline app/templates/menu-search.html]: action should make a request to the endpoint:
i.e
{{ url_for('routes.menu-browse') }}

it should also specify the request type to be POST

**Review by pgayee694 — CHANGES_REQUESTED** [review]

**Review by pgayee694 — APPROVED** [review]

**Pull request merged**